client/src/data.js

```diff
@@ -1,3 +1,16 @@
+export async function SigningUp(userData) {
```

Comment:
This isn't a component, so it should start with lowercase and should start with a verb (like 'signUp').